src/types.ts

```diff
@@ -615,6 +615,123 @@ export interface VisitRowsOptions {
   wrap?: boolean;
 }
 
+export type FilterOptionsType = {
+  /**
+   * Element or selector of an input control for filter query strings
+   * @default null
+   */
+  connectInput: null | string | Element;
```

Comment:
Does this work, even if the option is optional? Or should it rather be `connectInput?: ...`

Reply:
I made all props optional now